ui/src/components/LoginForm/SessionTable.jsx

```diff
@@ -1,9 +1,20 @@
 import React from 'react';
 import { Table } from 'react-bootstrap';
 import { LuExternalLink } from 'react-icons/lu';
-
 import SessionDateTime from './SessionDateTime';
 import styles from './SessionTable.module.css';
+import { isDateInRange } from './utils';
+
+const HIGHLIGHTED_TIME_RANGE = 1;
+
+function isAroundNow(session) {
+  const { actual_start_date, actual_end_date } = session;
+  return isDateInRange(
+    actual_start_date,
+    actual_end_date,
+    HIGHLIGHTED_TIME_RANGE,
+  );
+}
```

Comment:
My comment about not needing a generic `isDateInRange` still stands. In `utils.js`:

```js
function isAroundNow(session) {
  const startDate = parseDate(session.actual_start_date);
  const endDate = parseDate(session.actual_end_date);
  ...
  lowerBound.setDate(today.getDate() - 1); // or keep the `HIGHLIGHTED_TIME_RANGE` constant if you prefer, but use it here directly
  ...
  upperBound.setDate(today.getDate() + 1);
}
```